justfile

```diff
@@ -72,10 +72,6 @@ format:
   cargo fmt --all
   nixpkgs-fmt $(echo **.nix)
 
-# start tmuxinator with a dev federation setup
-tmuxinator:
-  ./scripts/tmuxinator.sh
-
-# exit tmuxinator session
-exit-tmuxinator:
-  tmux kill-session -t fedimint-dev
```

Comment:
note for exit fed-shell `mprocs --ctl '{c: quit}'`

Reply:
This command doesn't quite work since mprocs doesn't know the server address, so I think some more legwork is required for this

Reply:
Figured out how to specify server port for the mprocs session, so this is fixed